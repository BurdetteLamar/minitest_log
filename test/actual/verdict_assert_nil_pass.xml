<log>
  <summary verdicts='3' failures='0' errors='0'/>
  <section name='Test' method='verdict_assert_nil?' type='pass'>
    <section name='Args=[nil]'>
      <verdict method='verdict_assert_nil?' outcome='passed' id='0' message='Message 0'>
        <act_value>
          nil
        </act_value>
      </verdict>
    </section>
    <section name='Args=[nil]'>
      <verdict method='verdict_assert_nil?' outcome='passed' id='1' message='Message 1'>
        <act_value>
          nil
        </act_value>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
