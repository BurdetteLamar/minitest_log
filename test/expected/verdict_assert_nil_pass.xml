<log>
  <summary_ verdicts='3' failures='0' errors='0'/>
  <section_ name='Test' method='verdict_assert_nil?' type='pass'>
    <section_ name='Args=[nil, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_assert_nil?' outcome='passed' id='0' message='Message 0'>
        <actual_ class='NilClass' value='nil'/>
      </verdict_>
    </section_>
    <section_ name='Args=[nil, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_assert_nil?' outcome='passed' id='1' message='Message 1'>
        <actual_ class='NilClass' value='nil'/>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
